### DISCUSSION TITLE
Keeping the bot running on Windows

### DISCUSSION THREAD

**Discussion by socks5th** [Steffo99/greed#182] [Q&A]
I have the bot up and running on a Windows 10 VPS but id like to keep it running when I close my all my cmds. I'm not the best at coding and I know "systemd (recommended, but more complex)" is for Linux, so what would be the windows version of that? If I have greed installed on my desktop as a subfolder to the main on on my desktop?

**Comment by Steffo99** [comment]
I imagine the solution for Windows would be a system service, but I don't have the knowledge to give you instructions on how to proceed. 

Let's leave this open and see if somebody else has a solution!

**Reply by socks5th** [reply]
Okay, thanks for the reply! Right now Ive just been keeping the cmd open and minimized on the VPS instance and I haven't had any problems of course. So ill just continue doing that till I figure it out.
